main.py

```diff
@@ -51,6 +51,11 @@ def main():
                     df = data_management.add_transaction(df)
                 except UnboundLocalError or ValueError:
                     print("There is no file.")
+            case "5":
+                try:
+                    df = data_management.del_transaction(df)
```

Comment:
In current code, `df` will become `None` because `data_management.del_transaction` does not return value.